Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/KeyboardTypeModifier.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  KeyboardTypeModifier.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/18/23.
+//
+
+import SwiftUI
+
+/// Changes the capitalization of the text for any element.
+///
+/// ```html
+/// <TextField modifiers={keyboard_type(@native, type: :email_address)}>
+///     Email
+/// </TextField>
+/// ```
+///
+/// ## Arguments
+/// * ``type``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+struct KeyboardTypeModifier: ViewModifier, Decodable, Equatable {
+    /// One of the `UIKeyboardType` enumerations.
```

Comment:
Can you list the possible values?
```suggestion
    /// One of the `UIKeyboardType` enumerations.
    ///
    /// Possible values:
    /// * `default`
    /// * `ascii_capable`
    /// * `numbers_and_punctuation`
    /// * `url`
    /// * `number_pad`
    /// * `name_phone_pad`
    /// * `email_address`
    /// * `decimal_pad`
    /// * `twitter`
    /// * `web_search`
    /// * `ascii_capable_number_pad`
```